File: Archiver/frmNotifyMessage.vb
Public Class frmNotifyMessage

    Dim TimerCount As Integer = 0
    Public MsgToDisplay As String = ""

    Sub New()
        InitializeComponent()

        If (gNotifyMsg.Length.Equals(0)) Then
            Me.Close()
            Return
        End If
        txtMsg.Text = gNotifyMsg
        T1_Tick(Nothing, Nothing)
    End Sub

    Private Sub T1_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles T1.Tick
        TimerCount += 1

        txtMsg.Text = gNotifyMsg
        Me.Refresh()
        Application.DoEvents()

        If TimerCount >= 6 Then
            Me.Close()
        End If
    End Sub

End Class

File: Archiver/frmNotifyMessage.cs
using System;
using System.Windows.Forms;
using MODI;

namespace EcmArchiver
{
    public partial class frmNotifyMessage
    {
        private int TimerCount = 0;
        public string MsgToDisplay = "";

        public frmNotifyMessage()
        {
            InitializeComponent();
            if (modGlobals.gNotifyMsg.Length.Equals(0))
            {
                Close();
                return;
            }

            txtMsg.Text = modGlobals.gNotifyMsg;
            T1_Tick(null, null);
        }

        private void T1_Tick(object sender, EventArgs e)
        {
            TimerCount += 1;
            txtMsg.Text = modGlobals.gNotifyMsg;
            Refresh();
            Application.DoEvents();
            if (TimerCount >= 6)
            {
                Close();
            }
        }
    }
}